src/App.tsx

```diff
@@ -1,42 +1,38 @@
-import React from 'react';
+import { FC, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
-export const tabs = [
+import { Tab } from './types/Tab';
+import { Tabs } from './components/Tabs';
+
+export const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App: React.FC = () => {
+export const App: FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState<string>(tabs[0].id);
+
+  const handleSelectedItem = (tab: Tab) => {
+    setSelectedTabId(tab.id);
+  };
+
+  const findTab = (id: string): Tab | null => (
+    tabs.find(tab => tab.id === id) || null
+  );
```

Comment:
I guess, would be better to return the 1st tab is nothing found

Reply:
Done